doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-set-user-task-assignment-strategy.adoc

```diff
@@ -0,0 +1,30 @@
+[id='proc-set-user-task-assignment-strategy_{context}']
+
+= Setting the user task assignment strategy
+
+The user task assignment strategy is used to automatically assign the tasks to a suitable user. The assignment strategy allows more efficient task allocation based on the associated properties, such as potential owners, task priority, and task data. The default user task assignment strategy in Red Hat Process Automation Manager is `org.jbpm.task.assignment.strategy`. You can also explicitly define an assignment strategy for a user task in {CENTRAL}.
```

Comment:
Is this `org.jbpm.task.assignment.strategy` a system property or a strategy?

Reply:
Thanks @kaldesai. I've changed the sentence to "org.jbpm.task.assignment.strategy is the system property for the user task assignment strategy in {PRODUCT}"